cmd/cloud.go

```diff
@@ -0,0 +1,125 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2016 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/loadimpact/k6/stats/cloud"
+	"github.com/loadimpact/k6/ui"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+var cloudCmd = &cobra.Command{
+	Use:   "cloud",
+	Short: "Execute a test on the cloud",
+	Long: `Execute a test on the cloud.
+
+This will execute the test on the Load Impact cloud service. Use "k6 login cloud" to authenticate.`,
+	Example: `
+	k6 cloud script.js`[1:],
+	Args: cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		_, _ = BannerColor.Fprint(stdout, Banner+"\n\n")
+		fmt.Fprint(stdout, "  Deploying script to the cloud..")
```

Comment:
Hm. I'm iffy about the wording here; "Uploading" maybe? "Starting"?

"Deploying" makes me think of something more heavyweight than this; I "deploy" a website, for example.